travis-simple/.travis.yml

```diff
@@ -0,0 +1,58 @@
+#
+# DO NOT EDIT THIS FILE DIRECTLY
+#
+# It is automatically copied from https://github.com/pion/.goassets repository.
+# If this repository should have package specific CI config,
+# remove the repository name from .goassets/.github/workflows/assets-sync.yml.
+#
+
+language: go
+go:
+  - "1.x" # use the latest Go release
+
+branches:
+  only:
+  - master
+
+env:
+  global:
+    - GO111MODULE=on
+    - GOLANGCI_LINT_VERSION=1.18.0
+
+cache:
+  directories:
+    - $HOME/.cache/go-build
+    - $GOPATH/pkg/mod
+
+jobs:
+  include:
+    - name: lint
+      env: CACHE_NAME=lint
+      before_script:
+        - |
+          curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
+            | bash -s - -b $GOPATH/bin v${GOLANGCI_LINT_VERSION}
+      install: skip
+      script:
+        - bash .github/assert-contributors.sh
+        - bash .github/lint-disallowed-functions-in-library.sh
+        - bash .github/lint-commit-message.sh
+        - bash .github/lint-filename.sh
+        - golangci-lint run ./...
+    - name: test
+      env: CACHE_NAME=test
+      before_install:
+        - go mod download
+      install:
+        - go build ./...
+      script:
+        - coverpkgs=$(go list ./... | grep -v examples | paste -s -d ',')
+        - |
+          go test \
```

Comment:
If tests deadlock this doesn't produce any `stdout` it is really frustrating. 

I haven't looked at in a while, but I need to look if we can do this a better way.

Reply:
`go test -timeout=${TIMEOUT_SECONDS}s` may help?

Reply:
`go test -timeout=` specifies total time of the tests. It's not suitable for our case.